resources/user-agents/browsers/opera-mobile/opera-mobile-10-00.json

```diff
@@ -30,16 +30,19 @@
           "match": "Opera/9.80*(#PLATFORM#Opera Mobi*)*Version/#MAJORVER#.#MINORVER#*",
           "platforms": [
             "AndroidLinux",
-            "Android_4_0", "Android_4_1", "Android_4_2", "Android_4_3", "Android_4_4", "Android",
+            "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0",
+            "Android",
             "SymbianOS_C",
             "WinMobile"
           ]
         },
         {
           "match": "Opera/9.80*(#PLATFORM#Opera Tablet*)*Version/#MAJORVER#.#MINORVER#*",
+          "device": "general Tablet",
```

Comment:
I noticed this device was missing here, there was one test in issue-900 that touched this, so updated the test to be a Tablet device instead of a Mobile Phone.